### ISSUE TITLE
Can you individually provide the best-performing autoencoder model?

### ISSUE THREAD

**Issue by bo-wang-up** [danier97/LDMVFI#23]
Hi,

I trained the autoencoder myself but achieved unsatisfactory results. Could you please provide me with your best-performing autoencoder model? I would like to fine-tune it.

**Comment by danier97** [comment]
Hi there, thanks for your interest! I think this issue (#4) will help. 

Thanks.

**Comment by bo-wang-up** [comment]
Thanks! But how can I just load the autoendoer model in the main.py to fine tune the autoencoder? It seems that the autoencoder in evaluation is different from that in training

**Comment by bo-wang-up** [comment]
I notice that the pretrained model you provided does not include the parameters for loss.* in the autoencoder. Could you please provide a complete .ckpt file that includes these parameters? 
Thank you very much!

**Comment by Eduardo1000** [comment]
I'm also looking for the autoencoder. Thanks in advance!
